vuepress/docs/next/docs/reference/glossary.md

```diff
@@ -0,0 +1,75 @@
+| Entando Go Live Service | Provides expert help for issues arising in the final stages of a project, up to and including post-launch of a new Entando Application |
+| Entando Innovation Labs | Offers a multi-day session over which the customer and Entando build an initial application together to teach the customer Entando's methodology and platform |
+| Entando Upgrade Analysis | Reviews customer requirements and upgrade Entando to a later version |
+| Entando UX Standardization | Reviews methods to standardize UX across an enterprise |
+| Entando Web Application Security Analysis Service | Evaluates business vulnerabilities and designs a mitigation plan, in addition to solving known security issues |
+
+### Modules and Capabilities
+
+| Term |  Description
+|:--|:--
+| Entando Application | An application running on the Entando Platform and available at a given URL |
+| Entando App Builder | The frontend of the Entando Platform. Hosts the Entnado WCMS and provides the UI to interact with and configure Entando components. |
+| Entando App Engine | The core runtime engine for applications built with Entando App Builder. Provides the composition and rendering of the final application. |
+| Entando Bundle | A package built with Entando that contains one or more components |
+| Entando CLI (ent CLI) | Entando's core command line interface |
+| Entando Cloud Hub | A SaaS instance of an Entando Hub that contains a public and private collection of components |
+| Entando Component Generator (ECG) | An implementation of a JHipster blueprint that provides automation and templating to generate the components used to build an Entando Application |
+| Entando Component Manager (ECM) | Part of the ECR that coordinates the installation of the MFEs and MSs contained in an Entando Bundle |
+| Entando Hub |  A repository (local, remote, public, or private) containing Entando components. A single Entando App Builder can connect to 1 or more Entando Hubs. |
+| Entando Identity Management System | Entando's user management and authentication/authorization system, powered by Keycloak |
+| Entando Operator | Provides installation and application lifecycle automation for Entando Applications, microservices and required infrastructure services, e.g. databases and Keycloak |
+| Entando Web Content Management System (WCMS) | Entando's web content management system |
+| Local Hub | The local component repository in an Entando App Builder |
+| Entando Platform Capability (EPC) | A packaged capability that adds functionality to the platform and/or additional UX controls to the App Builder |
+| Entando Portal UI | 
+| Kubernetes integration service | Provides an abstraction layer between Entando microservices and the APIs exposed by a Kubernetes cluster |
+
+## General Technical Terms
+
+| Term |  Description
+|:--|:--
+| backend for frontend (BFF) | A software architecture pattern that connects the frontend of an application with the backend microservices |
```

Comment:
Definition feels a little off. A BFF microservice is itself a microservice dedicated to a frontend. It may also act as a facade to other enterprise microservices.